docs/user/commercial/privacy-level.rst

```diff
@@ -1,20 +1,41 @@
-.. TODO: This is a super weird page..
+Privacy Levels
+--------------
 
-:orphan:
+.. include:: /shared/admonition-rtd-business.rst
 
-Project privacy level
----------------------
+Privacy levels allow you to set the visibility of your project and its versions.
+This allows you to specify what information is available to the public and what is private.
+Privacy is generally controlled at the level of the version,
+and nothing more granular (eg. specific directories or URLs) can be controlled.
 
-.. include:: /shared/admonition-rtd-business.rst
+Project privacy
+~~~~~~~~~~~~~~~
 
 By default, only users that belong to your organization can see the dashboard of your project and its builds.
-If you want users outside your organization and anonymous users to be able to see the dashboard of your project,
+If you want users outside your organization to able to see the dashboard of your project,
 and the build output of *public versions* you can set the privacy level of your project to ``Public``.
 
-You can administer this in your :term:`dashboard` by navigating to :menuselection:`Admin --> Settings`
+You can set the project privacy level in your :term:`dashboard` by navigating to :menuselection:`Admin --> Settings`
 and changing :guilabel:`Privacy level` to `Public`.
 
-.. note::
+Version privacy
+~~~~~~~~~~~~~~~
+
+Each version of your documentation can be set to either `Public` or `Private`.
+This allows you to control who can see the documentation for a specific version.
+
+* Public versions are visible to everyone.
```

Comment:
Same here. I think we should specify this is "Documentation for public versions" -- since the access to the dashboard and builds of this version will be defined by "Project privacy"